app/components/works/edit/content_non_hierarchy_component.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render Elements::Tables::TableComponent.new(id: 'content-table', label: 'Files', show_label: false) do |component| %>
+    <%# These headers are duplicated in app/components/works/edit/content_hierarchy_component.html.erb %>
+    <% component.with_headers([
+                                { label: 'File Name' },
+                                { label: 'Description', tooltip: t('content_files.edit.fields.description.tooltip_html'), style: 'width: 180px;' },
+                                { label: 'Hide', tooltip: t('content_files.edit.fields.hide.tooltip_html'), style: 'width: 85px;' },
+                                { label: 'Action', style: 'width: 85px;' }
+                              ]) %>
+    <% content_files.each do |content_file| %>
+        <% component.with_row do |row_component| %>
+            <% row_component.with_cell do %>
+            <%= content_file.filepath %>
```

Comment:
Can you fix the indentation do that lines between do and end are indented?

Reply:
Fixed, will merge when PR is green.